public/index.html

```diff
@@ -19,8 +19,13 @@ <h1>TextSnippet</h1>
             <small></small>
         </div>
             <div class="button-container">
-                <button id="generate-snippet-btn">generate</button>
+                <button type="click" class="button" id="generate-snippet-btn">generate</button>
             </div>
+            <div class="muted">
+                <p id="output">speech muted</p>
+                <button  id="on" >on</button>
+            </div>
+                <button class="repeat" id="repeat" >read</button>
```

Comment:
give better id. Look at the generate button, it has "generate-snippet-btn"
So follow the same pattern and name it appropriately